cli/cmd/check.go

```diff
@@ -70,6 +66,7 @@ func (options *checkOptions) checkFlagSet() *pflag.FlagSet {
 	flags.StringVar(&options.cliVersionOverride, "cli-version-override", "", "Used to override the version of the cli (mostly for testing)")
 	flags.StringVarP(&options.output, "output", "o", options.output, "Output format. One of: basic, json")
 	flags.DurationVar(&options.wait, "wait", options.wait, "Maximum allowed time for all tests to pass")
+	flags.BoolVar(&options.short, "short", options.short, "Prints a check summary and only shows warnings and errors. Only for 'basic' output format")
```

Comment:
What do you think about having short mode attached to the `--output` flag instead of being its own flag?  i.e. `linkerd check -o short`.  Doing it that way means that we can avoid needing to do validation to prevent uses from doing invalid combinations like `linkerd check -o json --short`.

Reply:
Great idea!